simbev/trip.py

```diff
@@ -116,20 +116,40 @@ def create(self):
         self.fit_trip_to_timerange()
         self._set_timestamps()
 
+    def charge_decision(self, key):
+        return self.car.user_group.attractivity[key] > self.rng.random()
```

Comment:
can random become 1? if so it should be >=, since an attractivity of 1 needs to be 100%

Reply:
but on the other hand random can become zero und a attractivity of 0 can still lead to charging right? Or is random starting just over 0?

Reply:
Good point. It should basically never happen anyway.